src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -59,6 +59,7 @@ class ResourceType(Enum):  # pylint: disable=too-few-public-methods
     MGMT_RESOURCE_SUBSCRIPTIONS = ('azure.mgmt.resource.subscriptions', 'SubscriptionClient')
     MGMT_RESOURCE_DEPLOYMENTSCRIPTS = ('azure.mgmt.resource.deploymentscripts', 'DeploymentScriptsClient')
     MGMT_RESOURCE_TEMPLATESPECS = ('azure.mgmt.resource.templatespecs', 'TemplateSpecsClient')
+    MGMT_RESOURCE_DEPLOYMENTSTACKS = ('azure.mgmt.resource.deploymentstacks', 'DeploymentStacksClient')
```

Comment:
In fact, I did not find the folder of `azure.mgmt.resource.deploymentstacks` in package `azure.mgmt.resource`, even in the latest version [`21.2.0`](https://pypi.org/project/azure-mgmt-resource/21.2.0/)  
![Screenshot 2022-10-12 115556](https://user-images.githubusercontent.com/16612065/195274156-9451fe95-dc46-40c9-9ba7-ab2a25732887.png)
![Screenshot 2022-10-12 115840](https://user-images.githubusercontent.com/16612065/195274208-ca9616a6-6a77-4df5-b617-09acb8b69c2a.png)


Are you sure that `deploymentstacks` related content has been released in the SDK package `azure.mgmt.resource`?